config/authors/kodeflap.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Pooja Shaji",
+  "bio": "",
```

Comment:
Software Developer | Exploring Android Development|Learn and share knowledge

Reply:
Updated